compiler/plugins/target/LLVMCPU/LLVMIRPasses.cpp

```diff
@@ -56,7 +57,7 @@ LogicalResult runLLVMIRPasses(const LLVMTarget &target,
   case SanitizerKind::kAddress: {
     passBuilder.registerOptimizerLastEPCallback(
         [](llvm::ModulePassManager &modulePassManager,
-           llvm::OptimizationLevel Level) {
+           llvm::OptimizationLevel Level, llvm::ThinOrFullLTOPhase LTOPhase) {
```

Comment:
If this is unused, don't name the argument
```suggestion
           llvm::OptimizationLevel Level, llvm::ThinOrFullLTOPhase /*LTOPhase*/) {
```

Reply:
I'm just going to drop the arg name, I don't think it adds much.